django_celery_beat/models.py

```diff
@@ -220,7 +221,8 @@ class Meta:
         ordering = ['clocked_time']
 
     def __str__(self):
-        return '{}'.format(self.clocked_time)
+        return localtime(self.clocked_time).strftime("%d %B %Y %I:%M %p")
```

Comment:
can unit test added for this change?

Reply:
I think `return '{}'.format(make_aware(self.clocked_time))` is better.

Reply:
Yes @lvelvee I believe so. 
I will make another pull request with this change.

Reply:
Yes @lvelvee I believe so.
I have done another commit with this change after fetching the new base code.